src/ec/packages/ec-component-site-header/site-header.story.js

```diff
@@ -0,0 +1,74 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import englishBanner from '@ecl/ec-resources-logo/logo--en.svg';
+import frenchBanner from '@ecl/ec-resources-logo/logo--fr.svg';
+
+import siteHeaderDocs from './docs/site-header.md';
+import siteHeader from './site-header.html.twig';
+
+storiesOf('Components/Site Header', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'default',
+    () =>
+      siteHeader({
+        icon_file_path: defaultSprite,
+        language: {
+          url: '/example',
+          code: 'en',
+          label: 'English',
+        },
+        header_link: {
+          url: '/example-1',
+          aria_label: 'European Commission',
+        },
+        header_image: {
+          src: englishBanner,
+          alt: 'European Commission logo',
+          title: 'European Commission',
+        },
+        search_form: {
+          button: {
+            label: 'Search',
```

Comment:
Please add icon information here too (for english and translated). Currently the icon isn't displayed on mobile version.

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/33/commits/659a80d94c84289694dbd0bf95463b6209c291d4